corehq/apps/users/bulk_download.py

```diff
@@ -149,12 +194,28 @@ def _format_date(date):
         user_headers.extend(json_to_headers(
             {'location_code': list(range(1, max_location_length + 1))}
         ))
+    return user_headers, get_user_rows(user_dicts, user_headers)
 
-    def _user_rows():
-        for user_dict in user_dicts:
-            row = dict(flatten_json(user_dict))
-            yield [row.get(header) or '' for header in user_headers]
-    return user_headers, _user_rows()
+
+def parse_web_users(domain, task=None, total_count=None):
+    user_dicts = []
+    for n, user in enumerate(get_all_user_rows(domain, include_web_users=True, include_mobile_users=False,
+                                               include_inactive=False, include_docs=True)):
+        user_dict = make_web_user_dict(user, domain)
+        user_dicts.append(user_dict)
+        if task:
+            DownloadBase.set_progress(task, n, total_count)
+    for m, invite in enumerate(Invitation.by_domain(domain)):
+        user_dict = make_invited_web_user_dict(invite)
+        user_dicts.append(user_dict)
+        if task:
+            DownloadBase.set_progress(task, n + m, total_count)
+
+    user_headers = [
```

Comment:
It would be nice to order this more similarly to the mobile user one. Especially having username first, since that's the main identifier and guaranteed to always be there.

Reply:
Agreed that would be better